tests/main/regression-home-snap-root-owned/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Regression test that ensures that $HOME/snap is not root owned for sudo commands
+
+systems: [-ubuntu-core-16-64]
```

Comment:
I found the root cause and its an incomplete setup of the all-snap image for our tests. I fixed this now, thanks for the nudge :)